### PR TITLE
CLDR-15034 kbd: add mt

### PR DESCRIPTION
**Pull request by srl295**
CLDR-15034

- MSA 100:2002 is available from <https://mccaa.org.mt/Section/Content?contentId=1242>

Some references mentioning this layout:
- <http://web.archive.org/web/20160414071246/https://www.mita.gov.mt/MediaCenter/Images/1_Fonts_Pic1.jpg>
- <https://www.farah.cl/Keyboardery/A-Visual-Comparison-of-Different-National-Layouts/#mt>

ALLOW_MULTI_COMMITS=yes
ALLOW_MANY_COMMITS=yes

### CODE REVIEW

File: keyboards/3.0/mt.xml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+<keyboard locale="mt" conformsTo="techpreview">
+    <locales>
+        <!-- English is also an official language in Malta.-->
+        <locale id="en" />
+    </locales>
+    <info author="Steven R. Loomis" normalization="NFC" layout="QWERTY" indicator="MT" />
+
+    <names>
+        <name value="Maltese (48-key)" />
+        <!-- <name value="MSA 100:2002" /> -->
```
Comment (srl295) [inline keyboards/3.0/mt.xml:12]: multiple names not supported! will make separate PR

File: keyboards/3.0/mt.xml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        <key id="open-square" to="[" />
+        <key id="pipe" to="|" />
+        <key id="plus" to="+" />
+        <key id="pound" to="£" />
+        <key id="question" to="?" />
+        <key id="semi-colon" to=";" />
+        <key id="tilde" to="~" />
+        <key id="z-tikka" to="ż" />
+        <key id="Z-tikka" to="Ż" />
+    </keys>
```
Comment (srl295) [inline keyboards/3.0/mt.xml:66]: time to do some <imports> ..

File: keyboards/3.0/mt.xml [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+        <layer modifier="altR">
+            <row keys="grave gap gap pound gap gap gap gap gap gap gap gap gap" />
+            <row keys="gap gap e-grave gap gap gap u-grave i-grave o-grave gap open-square close-square" />
+            <row keys="a-grave gap gap gap gap gap gap gap gap gap gap gap" />
+            <row keys="backslash gap gap gap gap gap gap gap gap gap gap" />
+            <row keys="space" />
+        </layer>
+
+        <layer modifier="altR-shift">
```
Comment (srl295) [inline keyboards/3.0/mt.xml:93]: `altR+shift` is not supported by NMTOKEN. But we could fix that. added to agenda.

File: keyboards/3.0/mt.xml [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+        <layer modifier="none">
+            <row keys="c-tikka 1 2 3 4 5 6 7 8 9 0 minus equals" />
+            <row keys="q w e r t y u i o p g-tikka h-maqtua" />
+            <row keys="a s d f g h j k l semi-colon hash" />
+            <row keys="z-tikka z x c v b n m comma period slash" />
+            <row keys="space" />
+        </layer>
+
+        <layer modifier="shift">
+            <row keys="C-tikka bang double-quote euro dollar percent caret amp open-paren close-paren underscore plus " />
```
Comment (mcdurdin) [inline keyboards/3.0/mt.xml:78]:
```suggestion
            <row keys="C-tikka bang double-quote euro dollar percent caret amp open-paren close-paren underscore plus" />
```

Unimportant but tidy

File: keyboards/3.0/mt.xml [hunk 5]
```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE keyboard SYSTEM "../dtd/ldmlKeyboard.dtd">
+<keyboard locale="mt" conformsTo="techpreview">
+    <locales>
+        <!-- English is also an official language in Malta.-->
+        <locale id="en" />
+    </locales>
+    <info author="Steven R. Loomis" normalization="NFC" layout="QWERTY" indicator="MT" />
```
Comment (mcdurdin) [inline keyboards/3.0/mt.xml:8]: Authorship I hope will also be trackable by git commit / github PR. Because it's important to loop people in when we change things!

**Review by mcdurdin — APPROVED** [review]
LGTM. Noting that this is a hardware keyboard only. But I think it's proving the spec pretty well?

File: keyboards/3.0/mt.xml (hunk 5 above)
Comment (srl295) [reply keyboards/3.0/mt.xml:8]: definitely.  may want to add more metadata.  I should have put my id in with an email.
